distributed/comm/tcp.py

```diff
@@ -525,8 +528,10 @@ class TLSConnector(BaseTCPConnector):
     encrypted = True
 
     def _get_connect_args(self, **connection_args):
-        ctx = _expect_tls_context(connection_args)
-        return {"ssl_options": ctx}
+        tls_args = {"ssl_options": _expect_tls_context(connection_args)}
```

Comment:
I think this could be simplified now we're always going via start_tls

Reply:
Yeah, except I realized we can't always go via start_tls (just commented above).